packages/react-core/src/components/Select/examples/Select.md

```diff
@@ -3,35 +3,71 @@ id: Select
 section: components
 subsection: menus
 cssPrefix: pf-c-select
-propComponents: ['Select', 'SelectOption', 'SelectGroup', 'SelectList', 'MenuToggle']
+propComponents: ['Select', 'SelectOption', 'SelectGroup', 'SelectList', 'MenuToggle', 'SelectPopperProps']
 ouia: true
 ---
 
 import TimesIcon from '@patternfly/react-icons/dist/esm/icons/times-icon';
+import BellIcon from '@patternfly/react-icons/dist/esm/icons/bell-icon';
 
 ## Examples
 
 ### Single
 
 ```ts file="./SelectBasic.tsx"
+
+```
+
+### Option variations
```

Comment:
Sure I will update the description.

For the other feedback:
- Both single typeaheads behave this way currently, but I can update them to stop filtering after a selection (currently the examples use the input value to do filtering - the filter value just needs to be tracked in a separate state). 
- I added it there because Select's built-in `tab` behavior is incompatible with the example (`tab` needs to have its default behavior rather than close the select). Since this PR I updated Dropdown/Menu to allow customization of those closing keys, so I can add that update here as well and move it back into Select.

Reply:
The other reason for adding the demo to custom menus was because things like MenuSearch and MenuFooter lay outside the MenuContent, but in Select and Dropdown the MenuContent is internal so the extra children would be under the content instead of outside it. I think this is only a DOM difference though, without affecting keyboard or visual behavior.